fcrepo-clients/pom.xml

```diff
@@ -123,34 +123,27 @@
 	</build>
 
 	<dependencies>
+	    <!-- Logging -->
+        <dependency>
```

Comment:
Does this still need a log4j dependency? Doesn't seem to be in the other packages

Reply:
Don't think it has a log4j dependency?   slf4j is still used as an implementation independent api for logging.